units/systemd-journal-flush.service

```diff
@@ -20,6 +20,9 @@ Wants=systemd-journald.service
 After=systemd-journald.service
 RequiresMountsFor=/var/log/journal
 
+Before=soft-reboot.target systemd-soft-reboot.service
```

Comment:
systemd-journal-flush.service has After=systemd-journald.service, so it will be stopped before journald is stopped, and journald is stopped before soft-reboot is triggered, so this ordering doesn't seem necessary to me?

Reply:
Even if the `Before=` line is redundant, I'd like to add it to make the unit self-explanatory, like many units have both Conflicts= and Before=shutdown.target.